packages/mobile/src/identity/verification.ts

```diff
@@ -492,82 +489,68 @@ function* waitForAttestationCode(issuer: string) {
+}
+
+async function isAccountUpToDate(
+  accountsWrapper: AccountsWrapper,
+  address: string,
+  dataKey: string
+) {
+  const [currentWalletAddress, currentDEK] = await Promise.all([
+    accountsWrapper.getWalletAddress(address),
+    accountsWrapper.getDataEncryptionKey(address),
+  ])
+  return (
+    eqAddress(currentWalletAddress, address) && currentDEK && eqAddress(currentDEK.join(), dataKey)
+  )
 }
 
 export function* revokeVerification() {
   try {
     Logger.debug(TAG + '@revokeVerification', 'Revoking previous verification')
-    const account = yield call(getConnectedUnlockedAccount)
-    const [, e164NumberHash]: string[] = yield call(getE164NumberHash)
 
+    // TODO: https://github.com/celo-org/celo-monorepo/issues/1830
+    yield put(
+      showMessage('Revocation not currently supported. Just resetting local verification state')
+    )
     yield put(resetVerification())
-
-    const attestationsContract: AttestationsType = yield getAttestationsContract(web3)
-    const currentAccounts:
-      | string[]
-      | undefined = yield attestationsContract.methods
-      .lookupAccountsForIdentifier(e164NumberHash)
-      .call()
-    if (!currentAccounts) {
-      Logger.warn(TAG + '@revokeVerification', 'No accounts found for phone number')
-      return
-    }
-
-    const index = currentAccounts.map((a) => a.toLowerCase()).indexOf(account)
-    if (index < 0) {
-      Logger.warn(TAG + '@revokeVerification', 'Account not found')
-      return
-    }
-
-    const revokeTx = attestationsContract.methods.revoke(e164NumberHash, index)
-    yield call(sendTransaction, revokeTx, account, TAG, 'Revoke attestation')
-
     yield put(setNumberVerified(false))
-    Logger.debug(TAG + '@revokeVerification', 'Done revoking previous verification')
-    Logger.showMessage('Done revoking phone verification')
+
+    // const account = yield call(getConnectedUnlockedAccount)
+    // const [, e164NumberHash]: string[] = yield call(getE164NumberHash)
+    // const attestationsWrapper: AttestationsWrapper = yield getattestationsWrapper(web3)
+    // const currentAccounts:
+    //   | string[]
+    //   | undefined = yield attestationsWrapper.methods
+    //   .lookupAccountsForIdentifier(e164NumberHash)
+    //   .call()
+    // if (!currentAccounts) {
+    //   Logger.warn(TAG + '@revokeVerification', 'No accounts found for phone number')
+    //   return
+    // }
+
+    // const index = currentAccounts.map((a) => a.toLowerCase()).indexOf(account)
+    // if (index < 0) {
+    //   Logger.warn(TAG + '@revokeVerification', 'Account not found')
+    //   return
+    // }
+
+    // const revokeTx = attestationsWrapper.methods.revoke(e164NumberHash, index)
+    // yield call(sendTransaction, revokeTx, account, TAG, 'Revoke attestation')
+
+    // Logger.debug(TAG + '@revokeVerification', 'Done revoking previous verification')
+    // Logger.showMessage('Done revoking phone verification')
```

Comment:
Should we delete?

Reply:
We can. i expect the fix code would look similar to this so kept it for a reference. What would you prefer?

Reply:
Ok, I'm fine leaving this.